### ISSUE TITLE
Улучшения плагина рейтинга

### ISSUE THREAD

**Issue by georgehovh** [nb557/plugins#3]
Можем ли мы добавить функцию замены рейтинга TMDB на постерах и на странице фильма на рейтинг Кинопоиска?

**Comment by nb557** [comment]
Нет. Если речь про источник TMDB/CUB, то нет, так как это создаст кучу лишних запросов к API. На источника KP там и так рейтинги из Кинопоиска.
